astropy/io/misc/tests/test_hdf5.py

```diff
@@ -395,7 +401,7 @@ def test_preserve_serialized(tmpdir):
     assert t1.meta == t2.meta
 
 
-@pytest.mark.skipif('not HAS_H5PY')
+@pytest.mark.skipif('not HAS_H5PY' or 'not HAS_YAML')
```

Comment:
@astrofrog I meant it is possible to remove all strings: `not HAS_H5PY or not HAS_YAML`. But yes for now a string is fine.